src/applications/static-pages/tests/facilities/FacilityAppointmentWaitTimesWidget.unit.spec.jsx

```diff
@@ -50,9 +50,8 @@ describe('facilities <FacilityAppointmentWaitTimesWidget>', () => {
 
   it('should NOT render facility patient satisfaction score data', () => {
```

Comment:
Should we put this on a loop to check if the result is consistent - https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/testing/end-to-end/cypress-flaky-test-management.md#how-to-detect-flaky-tests ?

Reply:
We could, but I'm pretty confident that in this case, the issue was that we were modifying the test data on line 55 below, and that was causing issues for other tests.